lib/services/ui/chat/conversation_view_controller.dart

```diff
@@ -232,4 +233,27 @@ class ConversationViewController extends StatefulController with GetSingleTicker
     cm.setAllInactiveSync();
     Get.delete<ConversationViewController>(tag: tag);
   }
-}
\ No newline at end of file
+
+  Future<void> saveReplyToMessageState() async {
+    final prefs = await SharedPreferences.getInstance();
```

Comment:
Pretty sure we already have an instance of shared preferences in the settings service. Use `ss.prefs` to access it.